src/main/java/com/carrotcreative/cream/cache/CacheManager.java

```diff
@@ -40,6 +43,22 @@ private CacheManager(Context context)
         mRootDir = context.getCacheDir();
     }
 
+    public void setLruCacheSize(int lruCacheSize)
```

Comment:
Do you think perhaps `enableLruCache(int lruCacheSize)` would be more appropriate?

I think that this methods primary responsibility is to enable the cache